client/components/data/purchases/edit-card-details/index.jsx

```diff
@@ -35,6 +35,12 @@ const EditCardDetailsData = React.createClass( {
 		sites: React.PropTypes.object.isRequired
 	},
 
+	getDefaultProps() {
```

Comment:
Are we ever going to override the default `trackingSlug` by using the prop? If not, we should probably just store it as a property of the class and access it with `this.trackingSlug` in `purchasesMixin`.